aspnetcore/security/gdpr.md

```diff
@@ -57,15 +57,15 @@ This partial:
 
 ## Essential cookies
 
-If consent has not been given, only cookies marked essential are sent to the browser. The following code makes a cookie essential:
+If consent to store cookies hasn't been provided, only cookies marked essential are sent to the browser. The following code makes a cookie essential:
 
 [!code-csharp[Main](gdpr/sample/RP/Pages/Cookie.cshtml.cs?name=snippet1&highlight=5)]
 
 <a name="tempdata"></a>
 
-## Tempdata provider and session state cookies are not essential
+### TempData provider and session state cookies are not essential
```

Comment:
AFAIK we're using contractions in headings. It was just styles that we were avoiding, but the 🧀 may have moved on me.

```suggestion
### TempData provider and session state cookies aren't essential
```

Side-note: There are several unescaped underscores north of this causing the emphasis in this view. Recommend fixing those.